mlir/lib/Target/LLVMIR/Dialect/GENX/GENXToLLVMIRTranslation.cpp

```diff
@@ -189,6 +189,17 @@ static llvm::Value *createAtomicRMW(llvm::IRBuilderBase &builder,
                                   {ptr->getType(), val->getType()}, {ptr, val});
 }
 
+// Create a call to SPIR function for loading a joint matrix.
+static llvm::Value *
+createMatrixLoad(llvm::IRBuilderBase &builder, llvm::Value *res,
+                 llvm::Value *ptr, llvm::Value *stride,
+                 GENX::MatrixLayout layout, GENX::Scope scope,
+                 std::optional<GENX::MemoryAccess> memoryAccess) {
+  assert(isa<llvm::PointerType>(ptr->getType()) && "Expecting a pointer type");
+  assert(false && "TODO");
```

Comment:
Placeholder. @pengtu once you have added the ability to generate IGC "intrinsics" we can fill in this part.